core/src/test/java/com/google/errorprone/fixes/SuggestedFixesTest.java

```diff
@@ -846,27 +846,59 @@ public void qualifyStaticImport_whenAlreadyImported_usesUnqualifiedName() {
   }
 
   @Test
-  public void qualifyStaticImport_whenNamesClash_usesQualifiedName() {
+  public void qualifyStaticImport_whenIdentifierNamesClash_usesQualifiedName() {
     BugCheckerRefactoringTestHelper.newInstance(new ReplaceMethodInvocations(), getClass())
+        .addInputLines(
+            "pkg/Lib.java",
+            "package pkg;",
+            "public class Lib {",
+            "  public static void verifyNotNull(int a) {}",
+            " }")
+        .expectUnchanged()
         .addInputLines(
             "Test.java",
             "import static com.google.common.base.Preconditions.checkNotNull;",
+            "import static pkg.Lib.verifyNotNull;",
             "class Test {",
             "  void test() {",
             "    verifyNotNull(2);",
             "    checkNotNull(1);",
             "  }",
-            "  void verifyNotNull(int a) {}",
             "}")
         .addOutputLines(
             "Test.java",
             "import static com.google.common.base.Preconditions.checkNotNull;",
+            "import static pkg.Lib.verifyNotNull;",
             "import com.google.common.base.Verify;",
             "class Test {",
             "  void test() {",
             "    verifyNotNull(2);",
             "    Verify.verifyNotNull(1);",
             "  }",
+            "}")
+        .doTest();
+  }
+
+  @Test
+  public void qualifyStaticImport_whenMethodNamesClash_usesQualifiedName() {
```

Comment:
Prior to these changes this test would fail, because `verifyNotNull` is declared but not referenced.